### DISCUSSION TITLE
Homekit docker image for Synology

### DISCUSSION THREAD

**Discussion by Tiberius78** [tillsteinbach/VWsFriend#85] [Ideas]
Hi,

I've successfully added wvsfriend image to my Synology docker server and accessed to the car data. I've not yet figured out how to use the grafana image.

I'm interested to use the HomeKit part, but the HomeKit vwsfriend image doesn't seems to be available. I don't want to use docker-compose since it requires more skill to use on the Synology, I prefer to use the Synology docker UI.

<img width="1055" alt="Capture d’écran 2021-10-24 à 09 52 14" src="https://user-images.githubusercontent.com/93071782/138585546-7032b390-812b-4095-b9f1-269f21301945.png">

Could you provide a HomeKit image ? 

(If you want, if it works, I can then do a tutorial for Synology / HomeKit users)

**Comment by tillsteinbach** [comment]
You don't need a different Docker image. All you need is already inside the available one. You need to change the launch command to include --with-homekit you can see wharves necessary in the HomeKit compose files.
One important thing: It will only work if you are able to launch the container in host networking mode (or with a macvlan network) because homekit needs to be able to send out multicast messages.

**Reply by tillsteinbach** [reply]
Ahh, and a documentation how to make it work on the synology is very welcome of course!

**Reply by Tiberius78** [reply]
Thanks a lot ! I found how to add this parameter : in the ADDITIONAL_PARAMETERS environment variable and ... it works !
Ok, I'm going to write a tutorial now :)

**Comment by Tiberius78** [comment]
Hi,

I've finished a tutorial video (in French) for my YouTube chanel, I'll upload the video later.

I'ld like to update the wiki with a tutorial to install the container with Synology docker, how can I do that ?

**Reply by tillsteinbach** [reply]
Awesome! I changed the edit permissions for the repository. Can you check again?

**Comment by Tiberius78** [comment]
Ok, I've updated the Wiki :)

**Reply by tillsteinbach** [reply]
Big thanks for your contribution and the video is nice!

**Comment by upelectric** [comment]
I would like to continue the discussion because my topic goes in the same direction. I want to run the VWsfriend on my synology with homekit, database and grafana using a macvlan configuration. I used the macvlan.yml from Till. I changed it a bit to use bind mouns insteads of volumes but the default yml also runs. Unfortunately the HomeKit service is not available on the HOMEKIT_IP. Any suggestions? Currently I hardcoded the bind mount sources but it could be a good idea to use additional parameters in the env file for the bind mounts instead of it.
I will try to add them.

**Comment by Tiberius78** [comment]
FYI, I've added french subtitles to my tutorial, so the YouTube automatic translation should be more accurate.
